.github/workflows/helm.yml

```diff
@@ -0,0 +1,106 @@
+name: Helm
+on:
+  pull_request:
+    types: [edited, ready_for_review, opened, synchronize, reopened]
+  workflow_dispatch:
+
+jobs:
+  testing:
+    runs-on: ubuntu-latest
+    name: deploy and test
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: jitterbit/get-changed-files@v1
+      id: files
+      continue-on-error: true
+
+    - name: Run checks
+      id: check_helm_files
+      env:
+        PR_FILES_AM: ${{ steps.files.outputs.added_modified }}
+        PR_FILES_RENAMED: ${{ steps.files.outputs.renamed }}
+      run: |
+        PR_FILES="$PR_FILES_AM $PR_FILES_RENAMED"
+        for FILE in $PR_FILES; do
+            if [[ $FILE == helm-chart/* ]] ; then
+              echo "::set-output name=helm_dir_changed::true"
+              break
+            fi
+        done
```

Comment:
Let's move these two steps in separate job for example with name `check_changes`. And then job `testing` will have two additional configurations:
```yaml
  testing:
    needs: check_updates
    if:
      needs.check_updates.outputs.helm_dir_changed == 'true'
    ...  
```

Such way of configuration will rid condition `steps.check_helm_files.outputs.helm_dir_changed == 'true'` for each step.

Also you can see [nightly config](https://github.com/opencv/cvat/blob/develop/.github/workflows/schedule.yml) as example